### PR TITLE
New Random animation

### PR DESCRIPTION
**Pull request by rec**
A general purpose Random animation that you can control.

### CODE REVIEW

File: BiblioPixelAnimations/simple/Random.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    @classmethod
+    def get(cls, name):
+        dist = cls.DISTRIBUTIONS.get(name)
+        if dist:
+            return dist
+        else:
+            raise ValueError(NOT_DISTRIBUTION_FUNCTION_ERROR % name)
+
+
+_Distribution('betavariate', 256, alpha=1, beta=1)
```
Comment (adammhaile) [inline BiblioPixelAnimations/simple/Random.py]: Not a fan here... these are *always* going to get loaded even if you don't need them. Make these load on demand please.

**Comment by rec** [comment]
Why would you load this file if you weren't gonna use them?

It's not like constructing these costs anything significant!

**Comment by rec** [comment]
They're a table to look up reasonable starting parameters for the random number generators, and their names.

The moment you use this class, you have to create that whole table, because we look up the random number generator by name...

This is a tiny dictionary of tiny objects that cost almost nothing to construct, and it's in BPA, so no one will load it unless they're using it.  Likely the cost of loading layout/font.py is an order of magnitude greater than loading this file.

**Comment by adammhaile** [comment]
Fair +1.
Just doing my due diligence as I wasn't really sure what was going on.

**Pull request merged**